File: Go/01-两数之和.go
package main
func twoSum(nums []int, target int) []int {
	var res = []int{-1,-2}
	for i:=0;i<len(nums);i++ {
		temp := target - nums[i]
		for j:=i+1;j<len(nums);j++ {
			if temp == nums[j]{
				res[0]=j
				res[1]=i
				return res
			}
		}
	}
	return  res
}

File: Rust/01-两数之和.rs
impl Solution {
    pub fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32> {
        let mut rev =vec![-1i32;2];
        for i in 0..nums.len()  {
            let mut a = target - nums[i];
            for k in i+1..nums.len() {
                if a == nums[k] {
                    rev[0]=k as i32;
                    rev[1]=i as i32;
                    return rev 
                }
            }
        }
        rev
    }
}
